.github/workflows/scan.yaml

```diff
@@ -62,20 +62,23 @@ jobs:
   snyk-docker:
     name: Snyk Docker
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        docker-target: ["gitops", "gitops-server"]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Install stoml
-        run: make dependencies
       - name: Build docker image
-        run: make docker
+        env:
+          DOCKER_REGISTRY: ghcr.io/weaveworks
+        run: make docker-${{ matrix.docker-target }}
       - name: Scan docker image
         uses: snyk/actions/docker@master
         env:
           SNYK_TOKEN: ${{ secrets.SNYK_API_TOKEN }}
         with:
-          image: ghcr.io/weaveworks/wego-app:latest
-          args: --file=Dockerfile
+          image: ghcr.io/weaveworks/${{ matrix.docker-target }}:latest
```

Comment:
:question: Does this mean that we run these each of these matrices? So this job runs twice?

Reply:
Yes (although this is work done in the PR #1469 rather than here, will split & re-update)